docs/templates.rst

```diff
@@ -430,6 +430,36 @@ This gives back the results of the parent block::
     {% endblock %}
 
 
+Nesting template extends.
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In the case of multiple levels of %extends, the childmost {% block %} substitutes into the parent template. For instance,
```

Comment:
```suggestion
In the case of multiple levels of ``{% extends %}``,
``super`` references may be chained (as in ``super.super``)
to walk up the inheritance tree.

For example::
```

Reply:
> the childmost {% block %} substitutes into the parent template

- missing double-backticks around `{% blocks %}`
- I find the reference to "parent template" potentially unclear when there is a 3-level tree. Whose parent? If the childmost template's, s/the/its would clarify.
- Isn't the chainability of `super` references the motivation behind this PR / shouldn't it be mentioned here?